### ISSUE TITLE
SelectBox Custom: Cannot use first character to jump to item

### ISSUE THREAD

**Issue by cleidigh** [Microsoft/vscode#40540]
@bpasero 
@isidorn 
@Tyriar 
Including all who have commented :

New SelectBox implementation:

Standard native select allows navigation to an item using the first character.  This can be done both on a unexpanded or expanded select.

Should this behavior be implemented in the new custom SelectBox ?

**Comment by bpasero** [comment]
@cleidigh yeah good catch I think it should. If the control has focus at least.

**Comment by isidorn** [comment]
@cleidigh though assigning milestones is usualy a good idea, if you plan to fix this until the end of january just assign to December / January milestone.

**Comment by cleidigh** [comment]
@isidorn 
@bpasero 
Noted on the labeling issues (verified, candidate, milestones), sorry don't know all the workflows :-(

@bpasero Sorry to pepper you with a few more questions:
- I saw you wrote a draft test plan, would you like me to add some more specificity on keyboard support so people know exactly what to look for?
- If I have small tweaks/cleanup should I create an issue(s) or is using notes in a pull request sufficient?
Essentially when is it appropriate Not to do an issue for work in progress?

I will fix all issues posted for the December/January milestone assuming they are known in time.

**Comment by bpasero** [comment]
@cleidigh yeah feel free to make tweaks to the test plan item: https://github.com/Microsoft/vscode/issues/40589

I also published some release notes on it here: https://github.com/Microsoft/vscode-docs/blob/vnext/release-notes/v1_20.md#custom-dropdown-windows-linux

The advantage of having an issue is that it will get verified in the endgame if it is a bug. If the thing you are changing cannot be verified or is not a bug, doing just a PR is fine.

**Comment by bpasero** [comment]
@cleidigh no I think its fine

**Comment by cleidigh** [comment]
Fixed with: #40874
